[sdl2] Build error on x64-linux

Package: sdl2[core,wayland]:x64-linux@2.32.2

**Host Environment**

- Host: x64-linux
- Compiler: GNU 14.2.1
-    vcpkg-tool version: 2025-03-13-7699e411ea11543de6abc0c7d5fd11cfe0039ae5
    vcpkg-scripts version: 782ccc18d8 2025-03-15 (23 hours ago)

**To Reproduce**

`vcpkg install sdl2`

**Failure logs**

```
-- Using cached libsdl-org-SDL-release-2.32.2.tar.gz
-- Cleaning sources at openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source openblack/vcpkg/downloads/libsdl-org-SDL-release-2.32.2.tar.gz
-- Applying patch deps.patch
-- Applying patch alsa-dep-fix.patch
-- Applying patch cxx-linkage-pkgconfig.diff
-- Using source at openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean
CMake Warning at ports/sdl2/portfile.cmake:32 (message):
  You will need to install Wayland dependencies to use feature wayland:

  sudo apt install libwayland-dev libxkbcommon-dev libegl1-mesa-dev

Call Stack (most recent call first):
  scripts/ports.cmake:203 (include)


-- Found external ninja('1.12.1').
-- Configuring x64-linux
-- Building x64-linux-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /usr/bin/cmake --build . --config Debug --target install -- -v -j33
    Working Directory: openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg
    See logs for more information:
      openblack/vcpkg/buildtrees/sdl2/install-x64-linux-dbg-out.log

Call Stack (most recent call first):
  openblack/cmake-build-presets/ninja-multi-vcpkg/vcpkg_installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  openblack/cmake-build-presets/ninja-multi-vcpkg/vcpkg_installed/x64-linux/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/sdl2/portfile.cmake:58 (vcpkg_cmake_install)
  scripts/ports.cmake:203 (include)



```

<details><summary>openblack/vcpkg/buildtrees/sdl2/install-x64-linux-dbg-out.log</summary>

```
Change Dir: 'openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg'

Run Build Command(s): /usr/bin/ninja -v -v -j33 install
[1/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL.h
[2/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_assert.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_assert.h
[3/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_atomic.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_atomic.h
[4/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_audio.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_audio.h
[5/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_bits.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_bits.h
[6/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_blendmode.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_blendmode.h
[7/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_clipboard.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_clipboard.h
[8/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_copying.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_copying.h
[9/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_cpuinfo.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_cpuinfo.h
[10/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_egl.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_egl.h
[11/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_endian.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_endian.h
[12/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_error.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_error.h
[13/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_events.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_events.h
[14/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_filesystem.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_filesystem.h
[15/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_gamecontroller.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_gamecontroller.h
[16/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_gesture.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_gesture.h
[17/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_guid.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_guid.h
[18/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_haptic.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_haptic.h
[19/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_hidapi.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_hidapi.h
[20/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_hints.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_hints.h
[21/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_joystick.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_joystick.h
[22/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_keyboard.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_keyboard.h
[23/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_keycode.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_keycode.h
[24/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_loadso.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_loadso.h
[25/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_locale.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_locale.h
[26/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_log.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_log.h
[27/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_main.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_main.h
[28/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_metal.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_metal.h
[29/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_misc.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_misc.h
[30/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_messagebox.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_messagebox.h
[31/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_mouse.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_mouse.h
[32/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_mutex.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_mutex.h
[33/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_name.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_name.h
[34/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_opengl.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_opengl.h
[35/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_opengles2_gl2.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_opengles2_gl2.h
[36/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_opengles.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_opengles.h
[37/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_opengles2.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_opengles2.h
[38/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_opengles2_gl2ext.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_opengles2_gl2ext.h
[39/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_opengl_glext.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_opengl_glext.h
[40/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_opengles2_khrplatform.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_opengles2_khrplatform.h
[41/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_platform.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_platform.h
[42/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_opengles2_gl2platform.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_opengles2_gl2platform.h
[43/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_pixels.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_pixels.h
[44/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_power.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_power.h
[45/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_quit.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_quit.h
[46/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_rect.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_rect.h
[47/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_render.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_render.h
[48/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_rwops.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_rwops.h
[49/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_scancode.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_scancode.h
[50/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_sensor.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_sensor.h
[51/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_shape.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_shape.h
[52/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_stdinc.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_stdinc.h
[53/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_surface.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_surface.h
[54/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_system.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_system.h
[55/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_syswm.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_syswm.h
[56/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_test.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_test.h
[57/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_test_assert.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_test_assert.h
[58/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_test_common.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_test_common.h
[59/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_test_crc32.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_test_crc32.h
[60/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_test_fuzzer.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_test_fuzzer.h
[61/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_test_compare.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_test_compare.h
[62/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_test_font.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_test_font.h
[63/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_test_harness.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_test_harness.h
[64/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_test_images.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_test_images.h
[65/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/wayland-scanner client-header openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/wayland-protocols/cursor-shape-v1.xml openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols/cursor-shape-v1-client-protocol.h
[66/328] cd openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg && /usr/bin/cmake -E copy_if_different openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/include/SDL_test_log.h openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2/SDL_test_log.h
...
Skipped 233 lines
...
[300/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_ps5.c.o -MF CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_ps5.c.o.d -o CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_ps5.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/joystick/hidapi/SDL_hidapi_ps5.c
[301/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_rumble.c.o -MF CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_rumble.c.o.d -o CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_rumble.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/joystick/hidapi/SDL_hidapi_rumble.c
[302/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_shield.c.o -MF CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_shield.c.o.d -o CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_shield.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/joystick/hidapi/SDL_hidapi_shield.c
[303/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_stadia.c.o -MF CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_stadia.c.o.d -o CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_stadia.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/joystick/hidapi/SDL_hidapi_stadia.c
[304/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_steam.c.o -MF CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_steam.c.o.d -o CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_steam.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/joystick/hidapi/SDL_hidapi_steam.c
[305/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_steamdeck.c.o -MF CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_steamdeck.c.o.d -o CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_steamdeck.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/joystick/hidapi/SDL_hidapi_steamdeck.c
[306/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_switch.c.o -MF CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_switch.c.o.d -o CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_switch.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/joystick/hidapi/SDL_hidapi_switch.c
[307/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_wii.c.o -MF CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_wii.c.o.d -o CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_wii.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/joystick/hidapi/SDL_hidapi_wii.c
[308/328] : && /usr/bin/cmake -E rm -f libSDL2maind.a && /usr/bin/ar qc libSDL2maind.a  CMakeFiles/SDL2main.dir/src/main/dummy/SDL_dummy_main.c.o && /usr/bin/ranlib libSDL2maind.a && :
[309/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/sensor/dummy/SDL_dummysensor.c.o -MF CMakeFiles/SDL2-static.dir/src/sensor/dummy/SDL_dummysensor.c.o.d -o CMakeFiles/SDL2-static.dir/src/sensor/dummy/SDL_dummysensor.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/sensor/dummy/SDL_dummysensor.c
[310/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/thread/pthread/SDL_sysmutex.c.o -MF CMakeFiles/SDL2-static.dir/src/thread/pthread/SDL_sysmutex.c.o.d -o CMakeFiles/SDL2-static.dir/src/thread/pthread/SDL_sysmutex.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/thread/pthread/SDL_sysmutex.c
[311/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/joystick/steam/SDL_steamcontroller.c.o -MF CMakeFiles/SDL2-static.dir/src/joystick/steam/SDL_steamcontroller.c.o.d -o CMakeFiles/SDL2-static.dir/src/joystick/steam/SDL_steamcontroller.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/joystick/steam/SDL_steamcontroller.c
[312/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/thread/pthread/SDL_systls.c.o -MF CMakeFiles/SDL2-static.dir/src/thread/pthread/SDL_systls.c.o.d -o CMakeFiles/SDL2-static.dir/src/thread/pthread/SDL_systls.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/thread/pthread/SDL_systls.c
[313/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/locale/unix/SDL_syslocale.c.o -MF CMakeFiles/SDL2-static.dir/src/locale/unix/SDL_syslocale.c.o.d -o CMakeFiles/SDL2-static.dir/src/locale/unix/SDL_syslocale.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/locale/unix/SDL_syslocale.c
[314/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/filesystem/unix/SDL_sysfilesystem.c.o -MF CMakeFiles/SDL2-static.dir/src/filesystem/unix/SDL_sysfilesystem.c.o.d -o CMakeFiles/SDL2-static.dir/src/filesystem/unix/SDL_sysfilesystem.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/filesystem/unix/SDL_sysfilesystem.c
[315/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/thread/pthread/SDL_syscond.c.o -MF CMakeFiles/SDL2-static.dir/src/thread/pthread/SDL_syscond.c.o.d -o CMakeFiles/SDL2-static.dir/src/thread/pthread/SDL_syscond.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/thread/pthread/SDL_syscond.c
[316/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/timer/unix/SDL_systimer.c.o -MF CMakeFiles/SDL2-static.dir/src/timer/unix/SDL_systimer.c.o.d -o CMakeFiles/SDL2-static.dir/src/timer/unix/SDL_systimer.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/timer/unix/SDL_systimer.c
[317/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/power/linux/SDL_syspower.c.o -MF CMakeFiles/SDL2-static.dir/src/power/linux/SDL_syspower.c.o.d -o CMakeFiles/SDL2-static.dir/src/power/linux/SDL_syspower.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/power/linux/SDL_syspower.c
[318/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/misc/unix/SDL_sysurl.c.o -MF CMakeFiles/SDL2-static.dir/src/misc/unix/SDL_sysurl.c.o.d -o CMakeFiles/SDL2-static.dir/src/misc/unix/SDL_sysurl.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/misc/unix/SDL_sysurl.c
[319/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/thread/pthread/SDL_syssem.c.o -MF CMakeFiles/SDL2-static.dir/src/thread/pthread/SDL_syssem.c.o.d -o CMakeFiles/SDL2-static.dir/src/thread/pthread/SDL_syssem.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/thread/pthread/SDL_syssem.c
[320/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/thread/pthread/SDL_systhread.c.o -MF CMakeFiles/SDL2-static.dir/src/thread/pthread/SDL_systhread.c.o.d -o CMakeFiles/SDL2-static.dir/src/thread/pthread/SDL_systhread.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/thread/pthread/SDL_systhread.c
[321/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_xbox360.c.o -MF CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_xbox360.c.o.d -o CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_xbox360.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/joystick/hidapi/SDL_hidapi_xbox360.c
[322/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_xbox360w.c.o -MF CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_xbox360w.c.o.d -o CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_xbox360w.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/joystick/hidapi/SDL_hidapi_xbox360w.c
[323/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_xboxone.c.o -MF CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_xboxone.c.o.d -o CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapi_xboxone.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/joystick/hidapi/SDL_hidapi_xboxone.c
[324/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapijoystick.c.o -MF CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapijoystick.c.o.d -o CMakeFiles/SDL2-static.dir/src/joystick/hidapi/SDL_hidapijoystick.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/joystick/hidapi/SDL_hidapijoystick.c
[325/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/audio/pipewire/SDL_pipewire.c.o -MF CMakeFiles/SDL2-static.dir/src/audio/pipewire/SDL_pipewire.c.o.d -o CMakeFiles/SDL2-static.dir/src/audio/pipewire/SDL_pipewire.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/audio/pipewire/SDL_pipewire.c
FAILED: CMakeFiles/SDL2-static.dir/src/audio/pipewire/SDL_pipewire.c.o 
/usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/audio/pipewire/SDL_pipewire.c.o -MF CMakeFiles/SDL2-static.dir/src/audio/pipewire/SDL_pipewire.c.o.d -o CMakeFiles/SDL2-static.dir/src/audio/pipewire/SDL_pipewire.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/audio/pipewire/SDL_pipewire.c
openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/audio/pipewire/SDL_pipewire.c: In function ‘node_event_info’:
openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/audio/pipewire/SDL_pipewire.c:593:37: error: passing argument 1 of ‘pw_node_enum_params’ from incompatible pointer type [-Wincompatible-pointer-types]
  593 |             pw_node_enum_params(node->proxy, 0, info->params[i].id, 0, 0, NULL);
      |                                 ~~~~^~~~~~~
      |                                     |
      |                                     struct pw_proxy *
In file included from /usr/include/pipewire-0.3/pipewire/pipewire.h:30,
                 from openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/audio/pipewire/SDL_pipewire.h:28,
                 from openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/audio/pipewire/SDL_pipewire.c:29:
/usr/include/pipewire-0.3/pipewire/node.h:208:58: note: expected ‘struct pw_node *’ but argument is of type ‘struct pw_proxy *’
  208 | PW_API_NODE_IMPL int pw_node_enum_params(struct pw_node *object,
      |                                          ~~~~~~~~~~~~~~~~^~~~~~
[326/328] /usr/bin/ccache /usr/bin/cc -DSDL_STATIC_LIB -DUSING_GENERATED_CONFIG_H -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug/SDL2 -Iopenblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/include-config-debug -isystem /usr/include/libdrm -isystem openblack/vcpkg/buildtrees/sdl2/x64-linux-dbg/wayland-generated-protocols -isystem /usr/include/libdecor-0 -fPIC -idirafter "openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/video/khronos" -DHAVE_LINUX_VERSION_H  -Wall -Wundef -fno-strict-aliasing -Wdeclaration-after-statement -fvisibility=hidden -Wshadow -Wno-unused-local-typedefs -fdiagnostics-color=always -mmmx -msse -msse2 -msse3  -isystem/usr/include  -isystem/usr/include/pipewire-0.3  -isystem/usr/include/spa-0.2 -D_REENTRANT -DMESA_EGL_NO_X11_HEADERS -DEGL_NO_X11  -isystem/usr/include/libdrm  -DSDL_BUILD_MAJOR_VERSION=2 -DSDL_BUILD_MINOR_VERSION=32 -DSDL_BUILD_MICRO_VERSION=2 -g -MD -MT CMakeFiles/SDL2-static.dir/src/joystick/linux/SDL_sysjoystick.c.o -MF CMakeFiles/SDL2-static.dir/src/joystick/linux/SDL_sysjoystick.c.o.d -o CMakeFiles/SDL2-static.dir/src/joystick/linux/SDL_sysjoystick.c.o -c openblack/vcpkg/buildtrees/sdl2/src/ase-2.32.2-958db805cc.clean/src/joystick/linux/SDL_sysjoystick.c
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "name": "openblack",
  "version-string": "0.0.1-dev",
  "dependencies": [
    {
      "name": "sdl2",
      "default-features": true,
      "platform": "!linux"
    },
    {
      "name": "sdl2",
      "default-features": false,
      "features": [
        "wayland"
      ],
      "platform": "linux"
    },
    "imgui",
    "spdlog",
    "glm",
    "stb",
    "entt",
    "cxxopts",
    "openal-soft",
    "drlibs",
    {
      "name": "bgfx",
      "default-features": false,
      "features": [
        "tools",
        "multithreaded"
      ],
      "host": true,
      "$reason": "Don't want to build shaderc on non-native platforms so they are built on the host config."
    },
    {
      "name": "bgfx",
      "default-features": false,
      "platform": "emscripten"
    },
    {
      "name": "bgfx",
      "default-features": false,
      "features": [
        "multithreaded"
      ],
      "platform": "!emscripten"
    },
    "bullet3",
    "minizip",
    "gtest"
  ]
}

```
</details>

--- a/ports/sdl2/fix-pipewire.patch
+++ b/ports/sdl2/fix-pipewire.patch
@@ -1,0 +1,31 @@
+From 9e079fe9c7931738ed63d257b1d7fb8a07b66824 Mon Sep 17 00:00:00 2001
+From: Neal Gompa <neal@gompa.dev>
+Date: Mon, 10 Feb 2025 05:00:56 -0500
+Subject: [PATCH] pipewire: Ensure that the correct struct is used for
+ enumeration APIs
+
+PipeWire now requires the correct struct type is used, otherwise
+it will fail to compile.
+
+Reference: https://gitlab.freedesktop.org/pipewire/pipewire/-/commit/188d920733f0791413d3386e5536ee7377f71b2f
+
+Fixes: https://github.com/libsdl-org/SDL/issues/12224
+(cherry picked from commit d35bef64e913dd7d5dd3153a4b61f10ef837dad6)
+(cherry picked from commit 6be87ceb33a9aad3bf5204bb13b3a5e8b498fd26)
+---
+ src/audio/pipewire/SDL_pipewire.c | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/audio/pipewire/SDL_pipewire.c b/src/audio/pipewire/SDL_pipewire.c
+index 889e05decb293..5d1bfc28dedb7 100644
+--- a/src/audio/pipewire/SDL_pipewire.c
++++ b/src/audio/pipewire/SDL_pipewire.c
+@@ -590,7 +590,7 @@ static void node_event_info(void *object, const struct pw_node_info *info)
+ 
+         /* Need to parse the parameters to get the sample rate */
+         for (i = 0; i < info->n_params; ++i) {
+-            pw_node_enum_params(node->proxy, 0, info->params[i].id, 0, 0, NULL);
++            pw_node_enum_params((struct pw_node*)node->proxy, 0, info->params[i].id, 0, 0, NULL);
+         }
+ 
+         hotplug_core_sync(node);

--- a/ports/sdl2/portfile.cmake
+++ b/ports/sdl2/portfile.cmake
@@ -8,6 +8,7 @@ vcpkg_from_github(
         deps.patch
         alsa-dep-fix.patch
         cxx-linkage-pkgconfig.diff
+        fix-pipewire.patch
 )
 
 string(COMPARE EQUAL "${VCPKG_LIBRARY_LINKAGE}" "static" SDL_STATIC)

--- a/ports/sdl2/vcpkg.json
+++ b/ports/sdl2/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "sdl2",
   "version": "2.32.2",
+  "port-version": 1,
   "description": "Simple DirectMedia Layer is a cross-platform development library designed to provide low level access to audio, keyboard, mouse, joystick, and graphics hardware via OpenGL and Direct3D.",
   "homepage": "https://www.libsdl.org/download-2.0.php",
   "license": "Zlib",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -8358,7 +8358,7 @@
     },
     "sdl2": {
       "baseline": "2.32.2",
-      "port-version": 0
+      "port-version": 1
     },
     "sdl2-gfx": {
       "baseline": "1.0.4",

--- a/versions/s-/sdl2.json
+++ b/versions/s-/sdl2.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "8f5273484f07f8644858850d8942dd374535e96d",
+      "version": "2.32.2",
+      "port-version": 1
+    },
+    {
       "git-tree": "5fbe7b04157cab8298b2cafa18e786caf70c6ccd",
       "version": "2.32.2",
       "port-version": 0

